doc/changes/latest.inc

```diff
@@ -16,6 +16,7 @@ Current (0.22.dev0)
 
 Enhancements
 ~~~~~~~~~~~~
+- Add :func:`mne.read_evokeds_mff` to enable reading of averaged MFFs (requires mffpy library >= 0.5.5) by `Evan Hathaway`_ (:gh:`8354`)
```

Comment:
It looks like this is your first contribution (hooray)! Can you see how we've formatted `Eduard Ort`'s contribution? It creates a bold entry. You'll also need to add your name to `doc/changes/names.inc`, it's part of what's causing CircleCI to be unhappy.